### PR TITLE
Added Builtin Voltage

### PR DESCRIPTION
**Pull request by sadiqebrahim**
### Describe your change:

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms include at least one URL that points to Wikipedia or another similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: electronics/builtin_voltage.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
+from __future__ import annotations
+
+from math import log
+
+Q = 1.6021e-19  # ELECTRON CHARGE (units = C)
+K = 1.380649e-23  # BOLTZMAN'S CONSTANT (unit = kg m^2 s^-2 K^-1)
```
Comment (CaedenPH) [inline electronics/builtin_voltage.py:6]:
```suggestion
from scipy.constants import physical_constants, Boltzmann

Q = physical_constants['electron volt']  # ELECTRON CHARGE (units = C)
K = Boltzmann  # BOLTZMAN'S CONSTANT (unit = kg m^2 s^-2 K^-1)
```

File: electronics/builtin_voltage.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
+from __future__ import annotations
```
Comment (CaedenPH) [inline electronics/builtin_voltage.py:1]:
```suggestion
```
This isn't necessary here

**Review by CaedenPH — COMMENTED** [review]
Looks good - small changes to constants

File: electronics/builtin_voltage.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
+from math import log
+
+from scipy.constants import Boltzmann, physical_constants
+
+Q = physical_constants["electron volt"]  # ELECTRON CHARGE (units = C)
+K = Boltzmann  # BOLTZMAN'S CONSTANT (unit = kg m^2 s^-2 K^-1)
```
Comment (CaedenPH) [inline electronics/builtin_voltage.py:6]:
```suggestion
```
No point in making copies of the same const

File: electronics/builtin_voltage.py [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    elif donor_conc <= intrinsic_conc:
+        raise ValueError(
+            "Donor concentration should be greater than intrinsic concentration"
+        )
+    elif acceptor_conc <= intrinsic_conc:
+        raise ValueError(
+            "Acceptor concentration should be greater than intrinsic concentration"
+        )
+    else:
+        return K * T * log((donor_conc * acceptor_conc) / intrinsic_conc**2) / Q[0]
```
Comment (CaedenPH) [inline electronics/builtin_voltage.py:58]:
```suggestion
        return Boltzmann * T * log((donor_conc * acceptor_conc) / intrinsic_conc**2) / physical_constants["electron volt"][0]
```

**Review by CaedenPH — COMMENTED** [review]
My mistake -

**Review by CaedenPH — APPROVED** [review]
@cclauss Can you review pls

**Review by cclauss — APPROVED** [review]
Fixes hashes/elf.py

**Pull request merged**